create-projects/ocp-config/cd-jenkins/cd-jenkins-master.yml

```diff
@@ -19,11 +19,17 @@ parameters:
 - name: JENKINS_ENABLE_OAUTH
   value: "true"
 - name: JENKINS_MEMORY_LIMIT
-  value: 2Gi
+  value: 5Gi
```

Comment:
This won't fly. We run our cluster with a maximum 1:2 ratio for memory request/limit. And I assume many people do - as you should. If the ratio is to high, you're overcommitting the cluster too much, which is dangerous for memory.

Please change this to `4Gi` max. If you really need `5Gi` sometimes, we'd need to change the request to 2.5 or even 3 (the request should be close to what typical application usage is ...)

Reply:
Jenkins proposes to use all available memory for Heap space (we set this via `JENKINS_CONTAINER_HEAP_PERCENT` [here](https://github.com/opendevstack/ods-core/pull/431/files#diff-e52b91e71a932e8cd20b8dbd9e108f7cR31-R32)). When we set the `-Xmx` to 4GB [here](https://github.com/opendevstack/ods-core/pull/431/files#diff-e52b91e71a932e8cd20b8dbd9e108f7cR29-R30), we'll need to set the `JENKINS_MEMORY_LIMIT` to some value slightly higher (or reduce `JENKINS_CONTAINER_HEAP_PERCENT` to a lower value). I was not successful in setting `JENKINS_MEMORY_LIMIT` and `-Xmx` to the same value. In that case, Jenkins crashed. Will need to try this out.

Reply:
I see. What we definitely need to do though is to stay in the 1:2 ratio. I am wondering how you have been able to run this? If there is a cluster with a higher allowed ratio, that should be adjusted asap to avoid inconsistencies ...

Reply:
Got it. Will try out 4G and use a `JENKINS_CONTAINER_HEAP_PERCENT` of 0.95.